src/Client/index.ts

```diff
@@ -79,7 +79,7 @@ export type ConnectionTypeOptions =
   | GossipClusterOptions
   | SingleNodeOptions;
 
-interface ChannelCredentialOptions {
+export interface ChannelCredentialOptions {
```

Comment:
I agree these types should be public for convenience. But in that case, we should also make them available from the root, so you don't have to reference the file directly. 

Could you re-export them from `index.ts`.

Reply:
Sure, will do

Reply:
(By "them" I mean both the constructor arguments (`ChannelCredentialOptions` and `ConnectionTypeOptions`))